src/compareJ1939.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import pickle
+
+import pandas as pd
+
+
+def loadKnownIDs(filePath: str):
+    with open(filePath, "r") as f:
+        knownIDs: pd.DataFrame = pd.read_csv(f)
+        return knownIDs
+
+
+def importPickledData(filePath: str):
+    with open(filePath, "rb") as f:
+        loxamData: pd.DataFrame = pickle.load(f)
+        return loxamData
+
+
+def compareKnownIDs(knownIDs: list, loxamData: pd.DataFrame) -> list:
+    j1939Machines: list = []
+    for key in loxamData["Name"].unique():
+        matches = pd.merge(
+            knownIDs,
+            loxamData[loxamData["Name"] == key],
+            left_on="ID_HEX",
+            right_on="ID",
+            how="inner",
+        )
+        num_matches = len(matches)
+        if (
+            len(loxamData[loxamData["Name"] == key])
+            / (len(loxamData[loxamData["Name"] == key]) - num_matches)
+            > 1.2
+        ):
+            j1939Machines.append(key)
+        print(f"{key} has {num_matches} matches")
+    print(f"J1939 machines: {j1939Machines}")
+    print(loxamData[loxamData["Name"] == "weidemann 2070LP"])
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        prog="compareJ1939", description="Compares known J1939 IDs to loxam data"
+    )
+
+    parser.add_argument("folder1")
```

Comment:
evt. et lidt mere sigende navn i stedet for folder1 og folder2 evt. inputCanFolder eller lign. Jeg ville nok kigge på der for at vide hvad det er for en mappe.